Modules/xASL_module_ASL.m

```diff
@@ -323,7 +348,7 @@
             xASL_im_CenterOfMass(x.P.Path_ASL4D, OtherList, 10); % set CenterOfMass to lower accepted distance for when rerunning wrong registration
         end
 
-        if  nVolumes>1
+        if  nVolumes>1 && ~x.modules.asl.bContainsDeltaM && (~isfield(x.Q,'LookLocker') || (x.Q.LookLocker == 0))
             % Run motion Correction
             xASL_wrp_RealignASL(x);
```

Comment:
Here we do check for `bContainsDeltaM`, but to be sure I would also do this inside `xASL_wrp_RealignASL`, only once, at the top of the wrapper where the admin is done. We can remove it elsewhere in `xASL_wrp_RealignASL`.

Reply:
I'll quickly try to implement what you want, but this isn't how reviewing should be done. We shouldn't stack feature request over feature request again and again.

Reply:
9256d4e

Reply:
See all my comments above. `bContainsDeltaM` by itself is not a reason not to run motion correction. It's a combination with other factors that disable or modify motion correction. So warning removed. And removed `bContainsDeltaM` from this if-statement - see explanations above.

Reply:
See above. Let's replace this by `bMotionCorrection` and `bZigzag` et, which we all parse nicely in the header. Then it's easier to discuss what needs to happen and you don't have to read the whole code to understand this.

Reply:
Makes no sense to me. We moved all the details inside wrappers to keep ASL-module clean. It doesn't make sense to start moving things back from wrappers again.

`bContainsDeltaM` is now removed from here. And we can remove `nVolumes` and `LookLocker` and put it inside the wrapper if you want. But any larger changes do not make sense here.

Reply:
Not outside the wrapper, just to the header of the wrapper

Reply:
I see. Sorry, missed that. Yes, we can move that in.

Reply:
Fixed 791df878